tools/dptools/src/dptools/scripts/dp_dos.py

```diff
@@ -71,7 +71,8 @@ def parse_arguments(cmdlineargs=None):
         cmdlineargs: List of command line arguments. When None, arguments in
             sys.argv are parsed (Default: None).
     '''
-    parser = argparse.ArgumentParser(description=USAGE)
+    parser = ArgumentParser(
+        description=USAGE, formatter_class=RawTextHelpFormatter)
```

Comment:
Is there any reason to use the `RawTextHelpFormatter` class? We would then have to format each option ourself, it won't be adaptive to the terminal width, etc.

Reply:
I used the `RawTextHelpFormatter` class to format the `--auto-align` option, so that there is a visual distinction between the integer and fractional occupation case.

Reply:
I've undone the switch to `RawTextHelpFormatter` (also for `dp_bands`, which has already been merged). Ready to go from my side.